pkg/apis/pipeline/v1alpha1/artifact_pvc.go

```diff
@@ -67,19 +67,19 @@ func (p *ArtifactPVC) GetCopyToStorageFromContainerSpec(name, sourcePath, destin
 
 			"-args", strings.Join([]string{"mkdir", "-p", destinationPath}, " "),
 		},
-		VolumeMounts: []corev1.VolumeMount{getPvcMount(p.Name)},
+		VolumeMounts: []corev1.VolumeMount{GetPvcMount(p.Name)},
 	}, {
 		Name:    names.SimpleNameGenerator.RestrictLengthWithRandomSuffix(fmt.Sprintf("source-copy-%s", name)),
 		Image:   *BashNoopImage,
 		Command: []string{"/ko-app/bash"},
 		Args: []string{
 			"-args", strings.Join([]string{"cp", "-r", fmt.Sprintf("%s/.", sourcePath), destinationPath}, " "),
 		},
-		VolumeMounts: []corev1.VolumeMount{getPvcMount(p.Name)},
+		VolumeMounts: []corev1.VolumeMount{GetPvcMount(p.Name)},
 	}}
 }
 
-func getPvcMount(name string) corev1.VolumeMount {
+func GetPvcMount(name string) corev1.VolumeMount {
```

Comment:
As it is exported, we should add a comment on what it's doing :angel:

Reply:
Done. Thx.